master/buildbot/buildslave/openstack.py

```diff
@@ -16,14 +16,14 @@
 # This module is left for backward compatibility of old-named worker API.
 # It should never be imported by Buildbot.
 
+from buildbot.worker.openstack import OpenStackLatentWorker as _OpenStackLatentWorker
 from buildbot.worker_transition import deprecatedWorkerModuleAttribute
 from buildbot.worker_transition import reportDeprecatedWorkerModuleUsage
 
 reportDeprecatedWorkerModuleUsage(
     "'{old}' module is deprecated, use "
     "'buildbot.worker.openstack' module instead".format(old=__name__))
 
-from buildbot.worker.openstack import OpenStackLatentWorker as _OpenStackLatentWorker
 
```

Comment:
are you sure?
No unit tests are failing because of that

I would prefer to have a tested solution for that, and compatible with import sorting